x-pack/plugins/ml/kibana.json

```diff
@@ -16,7 +16,8 @@
     "share",
     "embeddable",
     "uiActions",
-    "kibanaLegacy"
+    "kibanaLegacy",
+    "indexPatternManagement"
```

Comment:
nit: This probably doesn't _have_ to be a required plugin, but since there's already a hard requirement on the `data` plugin which houses the index patterns service, it probably doesn't make a difference whether it's required or optional.